modules/layers/src/text-layer/text-background-layer/text-background-layer-vertex.glsl.js

```diff
@@ -46,7 +47,7 @@ void main(void) {
 
   // project meters to pixels and clamp to limits
   float sizePixels = clamp(
-    project_size_to_pixel(instanceSizes * sizeScale),
+    pixelSize ? instanceSizes * sizeScale : project_size_to_pixel(instanceSizes * sizeScale),
```

Comment:
Would it make sense to add a pixelSize argument to the `project function` and remove the if statement in all the shaders?